kube/objects.go

```diff
@@ -73,67 +73,67 @@ func (c *Client) FetchObjects(ctx context.Context, filter ObjectFilter) (*Object
 	var g errgroup.Group
 
 	g.Go(func() (err error) {
-		objects.Nodes, err = client.Nodes().List(opts)
+		objects.Nodes, err = client.Nodes().List(ctx, opts)
```

Comment:
Now that we can pass a context here, we could use `errgroup.WithContext` to cancel any remaining calls if any one call fails. That would let us fail faster in error cases.

Reply:
This is with 1.18, so, when I update to 1.18, I'll make sure to do this